### PR TITLE
No code quickstart

### PR DESCRIPTION
**Pull request by stb13579**
To complement the in-app instructions and help users find no-code through the documentation.

### CODE REVIEW

**Review by abarreau — APPROVED** [review]

File: content/tutorials/no-code/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Once you start your simulation, the load testing data are displayed in real-time. The following guide assists you in writing and launching your first load test. To keep learning about Gatling and load testing, see these helpful resources:
+- [Gatling Community](https://community.gatling.io)
+- [Gatling Academy](https://gatling.io/academy/)
+
+## Access Gatling Enterprise Cloud
+To access the Gatling no-code generator:
+1. Navigate to https://cloud.gatling.io in your web browser.
+2. Login or register if you don't have an account. 
+3. Click on **Create a new NO CODE simulation** in the _Simulation runs_ pane on the landing page. <!--TO BE Confirmed-->
```
Comment (stb13579) [inline content/tutorials/no-code/index.md:32]:
```suggestion
3. Click on **Create a simulation without coding** in the _Latest simulation runs_ pane on the landing page. 
```

Comment (stb13579) [reply content/tutorials/no-code/index.md:32]: updated from the testing hour version today

**Review by tpetillot — APPROVED** [review]

File: content/tutorials/no-code/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- [Gatling Community](https://community.gatling.io)
+- [Gatling Academy](https://gatling.io/academy/)
+
+## Access Gatling Enterprise Cloud
+To access the Gatling no-code generator:
+1. Navigate to https://cloud.gatling.io in your web browser.
+2. Login or register if you don't have an account. 
+3. Click on **Create a simulation without coding** in the _Latest simulation runs_ pane on the landing page. 
+
+The no-code generator is divided into 4 steps with a dynamically updated code example in Java, Scala, and Kotlin. The code example lets you see how changes in the simulation settings can be written in your preferred programming language.
```
Comment (Obero) [inline content/tutorials/no-code/index.md:34]: The phrasing seems a bit weird to me ?
Suggestion:

```suggestion
The no-code generator is divided into 4 steps with a dynamically updated code example in Java, Scala, and Kotlin. The code example shows you how the resulting simulation can be written in your preferred programming language.  
```

**Review by Obero — APPROVED** [review]
Nice doc ! Maybe consider adding some screenshots ? Or maybe the UI is meant to evolve quickly and this would be too costly to maintain ?

**Comment by stb13579** [comment]
> Nice doc ! Maybe consider adding some screenshots ? Or maybe the UI is meant to evolve quickly and this would be too costly to maintain ?

Good point! I added screenshots, except for the beginning since the user homepage isn't completed.

**Review by Obero — APPROVED** [review]

**Pull request merged**